Source/WebCore/platform/graphics/GraphicsContextGL.h

```diff
@@ -28,7 +28,9 @@
 #if ENABLE(WEBGL)
 
 #include "DestinationColorSpace.h"
+#include "GraphicsContextGLActiveInfo.h"
```

Comment:
It looks like GraphicsContextGLActiveInfo can get forward declared rather than included here.

Reply:
I've included it here to make `GeneratedSerializers.mm` compile because I didn't want to learn how `GeneratedSerializers.mm` is generated. I've investigated the generator and implemented it properly now.